index.coffee

```diff
@@ -214,12 +208,13 @@ exports.DockerProgress = class DockerProgress
 
 	# Create a stream that transforms `docker.modem.followProgress` onProgress events to include total progress metrics.
 	pullProgress: (image, onProgress) ->
+		progressRenderer = @renderProgress.bind(this)
```

Comment:
You can use `renderProgress: (percentage, completedSize, totalSize, width = PROGRESS_BAR_WIDTH) =>` (fat arrow) to avoid needing to bind here